treeherder/etl/perf.py

```diff
@@ -37,6 +38,10 @@ def _get_signature_hash(signature_properties):
     return sha.hexdigest()
 
 
+def _order_and_concat(words: List) -> str:
+    return ' '.join(sorted(words))
```

Comment:
lgtm.
There is a small risk that the limit (422 for extra_options/360 for tags) to be exceeded, thus having a IntegrityError (or something like this from Django DB).
Even so, I think it is better to put that empty space between list's elements.

Reply:
I don't think we need to worry about this. Before this function is called, the client code performs a schema validation, which eliminates this risk.